### DISCUSSION TITLE
Best definition to use for the EDLQ07CAV3 monoblock

### DISCUSSION THREAD

**Discussion by jbinko** [raomin/ESPAltherma#242] [General]
For this https://www.daikin.co.uk/content/internet/internet-uk/en_GB/products/product.html/EBLQ-CV3.html is the best https://github.com/raomin/ESPAltherma/blob/main/include/def/ALTHERMA(MONOBLOC_CA_05-07KW).h ? Not sure how to select the best header file.

**Comment by bashers222** [comment]
This one worked for me
